scripts/nearest-color

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
```

Comment:
What is this used for?

Reply:
This isn't really very important - I wrote this when I was extracting the palettes and didn't want to entirely throw the code away.

(I didn't want to exactly steal palettes from other sites, so I took palettes that existed and converted them to the nearest named colors.)

I'd be very willing to drop this, but my plan was actually to remove `bp color` (because I'm wanting to do less in `bp`) and merge it into this.

Reply:
ok.... I just wanted to make sure it was clear what this script was for. Please at least add a usage note (include why you would use it). But fine with keeping it.